src/components/Checkbox/index.tsx

```diff
@@ -46,24 +56,36 @@ const StyledIcon = styled(Icon)`
 `
 
 const StyledChildrenContainer = styled('div', {
-  shouldForwardProp: prop => !['size'].includes(prop),
+  shouldForwardProp: (prop: string) => !['size'].includes(prop),
```

Comment:
```suggestion
  shouldForwardProp: prop => !['size'].includes(prop.toString()),
```

prop is a PropertyKey (string | number | symbol) and they all have a toString method :)

Reply:
Thanks !